private/lib/coordinates.bzl

```diff
@@ -5,7 +5,7 @@ def unpack_coordinates(coords):
 
     Assumes `coords` is in one of the following syntaxes:
      * groupId:artifactId[:type[:scope]]:version
-     * groupId:artifactId[:version][@classifier][:type]
+     * groupId:artifactId[:version][:classifier][@type]
```

Comment:
I think this was a typo according to https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.dsl.DependencyHandler.html#N1737E